src/main/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -85,8 +85,8 @@
     private static final Log log = Log.getInstance(CollectInsertSizeMetrics.class);
     protected static final String Histogram_R_SCRIPT = "picard/analysis/insertSizeHistogram.R";
 
-    @Argument(shortName="H", doc="File to write insert size Histogram chart to.")
-    public File Histogram_FILE;
+    @Argument(shortName = "H", doc = "File to write insert size Histogram chart to.", optional = true)
```

Comment:
While we are here, could you make it clear that the output will be a PDF? Also, please make it clear in the doc that failing to specify the PDF will result in no PDF output (and a warning)